.ci/gitlab/common.yml

```diff
@@ -0,0 +1,29 @@
+.common:
+  image: docker.pkg.github.com/clash-lang/clash-compiler/clash-ci-$GHC_VERSION:2021-01-11
+  timeout: 2 hours
+  stage: build
+  variables:
+    GIT_SUBMODULE_STRATEGY: recursive
+    TERM: xterm-color
```

Comment:
Ah right, so presumably we can reliably expect ANSI colour support from runners anyway. Perhaps the previous comment about colour support isn't needed then

Reply:
Right, I think we should be able to! 

This might very well have been a leftover from our Circle/Travis days though.